File: java64/nivel5/exer53.java
import javax.swing.JOptionPane;

public class exer53 {
    public static void main(String[] args) {
        int melhorTempo = 0, tempoMedio, soma = 0, melhorVolta = 0;
        int voltas = Integer.parseInt(JOptionPane.showInputDialog("Número de voltas"));
        int tempo[] = new int[voltas];

        for (int i = 0; i < tempo.length; i++) {
            tempo[i] = Integer.parseInt(JOptionPane.showInputDialog("Informe o tempo da " + (i + 1) + " volta"));
            soma += tempo[i];
            if (tempo[i] < melhorTempo || i == 0) {
                melhorTempo = tempo[i];
                melhorVolta = i + 1;
            }
        }

        tempoMedio = soma / voltas;
        JOptionPane.showMessageDialog(null, "Melhor volta: " + melhorVolta + "\nMelhor tempo: " + melhorTempo + "\nTempo médio: " + tempoMedio);
    }
} 

File: pyhton64/nivel5/exer53.py
import string


n = int(input('Número de voltas: '))
voltas = []

for i in range(1, n + 1):
    voltas.append(float(input('Tempo das voltas: ')))

melhor_tempo = min(voltas)
melhor_volta = voltas.index(melhor_tempo)
melhor_volta += 1

media = sum(voltas) / n
print(f'Melhor tempo {melhor_tempo}\n na volta {melhor_volta}\nMédia de voltas: {media}')